Expose slinkity's intermediateDir to a `process.env` variable (#206)
slinkity/slinkity: www/src/docs/asset-management.md

```

Note that the `public` directory should be **at the root level of your passthrough copy**.

## Integrating Eleventy Image

> If you’re using the [Eleventy Image](https://www.11ty.dev/docs/plugins/image/) plugin to generate multiple formats, sizes or locally cached remote images you’ll need to configure it in a slinkity specific way. Slinkity generates an intermediate build folder (eg. `.11ty-build-DcNVBN`) and exposes this folder in `process.env.ELEVENTY_BUILD_OUTPUT_DIR` which you can use to generate the images to, as per:

Reviewer: ```suggestion
If you’re using the [Eleventy Image](https://www.11ty.dev/docs/plugins/image/) plugin to generate multiple formats, sizes or locally cached remote images, you’ll need to configure the build output in a Slinkity-specific way. Slinkity generates an intermediate build folder (eg. `.11ty-build-DcNVBN`) and exposes the path on an environment variable: `process.env.ELEVENTY_BUILD_OUTPUT_DIR`. You can use this as a destination for 11ty image and anything else that outputs to the build directory. For example:
```

Nit: Tweak some wording and mention this works for _anything_ outputting to the builder folder.

> ```js

Reviewer: ```suggestion
```js
const path = require('path')

```

Super pedantic, but I like to include global imports for those unaware of them!

await Image(src, {
  widths: [300, 600],
  formats: ["avif", "jpeg"],
  outputDir: path.join(process.env.ELEVENTY_BUILD_OUTPUT_DIR, 'img')
});
```

> For further Eleventy Image configuration you can check out their [excellent docs](https://www.11ty.dev/docs/plugins/image/).

Reviewer: ```suggestion
For further Eleventy Image configuration, you can check out their [excellent docs](https://www.11ty.dev/docs/plugins/image/).
```

Nit: I like commas
